### ISSUE TITLE
The theme preview images don't line up perfectly with the box ontop of them

### ISSUE THREAD

**Issue by vrman123** [suchmememanyskill/SDH-CssLoader#8]
very small issue, only visible on brightly coloured preview images.
it's more visible on the steam deck, these screenshots were taken from PC so there's some compression.

![image](https://user-images.githubusercontent.com/48649272/180437804-163486a7-755a-487a-80b5-527106c1c4ed.png)
![image](https://user-images.githubusercontent.com/48649272/180438051-5628a095-6695-4ee8-8977-6e0385c78dbf.png)

The gray box that appears when hovering is square instead of rounded (may be intentional?) but is also is slightly too high and doesn't cover the whole bottom of the box
![image](https://user-images.githubusercontent.com/48649272/180438508-2c6e23b3-d6f5-41b8-9fa8-dbcea1539397.png)

**Comment by beebls** [comment]
The "gray box" being not rounded is out of my control as it is a component from the decky-frontend-lib and I cannot modify it.

In PR #10, I adjusted the margin/padding of the install button to fix the small clipping issues you showed in the last picture.

As for the corners "peeking out", it is quite weird as the overlay layer has the exact size and rounding of the bottom image layer, so I feel like it is some kind of rendering bug, although I'm no HTML wizard so I might be wrong. For a temporary fix, I adjusted the rounding of the overlay layer to make it extend over those corners.

Once PR #10 is merged, all of these will be fixed.
